_studio/hevce_hw/h265/include/mfx_h265_encode_hw_utils.h

```diff
@@ -800,6 +800,7 @@ class MfxVideoParam : public mfxVideoParam, public TemporalLayers
     bool   bMBQPInput;
     bool   RAPIntra;
     bool   bFieldReord;
+    bool   bNonStandardReord; // Possible in encoding order only. No NumRefFrames limitation.
```

Comment:
Why it is not the same as EncodingOrder? In EncodingOrder the order parameters verification should be minimal, I think.